doc/DEMO_TESTNET.md

```diff
@@ -28,12 +28,12 @@ This tutorial has been tested on Linux, it *might* work on Unix and it won't wor
 Please refer to [practical-revault](https://github.com/revault/practical-revault/) for in-depth explanation of the architecture.
 
 Every stakeholder will need:
-- `revaultd`
+
```

Comment:
Maybe this should be updated if we bump the revault-gui version as well, no? Otherwise in revault-gui 0.2 you still need to run revaultd

Reply:
I should move the doc commit to another PR then and We will update it for next release